yevhenii.dolmatov/task_05/csv.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+def parse_csv_data(file)
+  CSV.parse(File.read(file), headers: true)
+end
+
+def parse_to_int(data)
+  CSV.parse(data.join(', '), converters: :numeric).flatten
+end
+
+def month_dates_length(dates_arr, regexp)
+  dates_arr.select { |d| d.match(regexp) }.length
+end
+
+def month_temperature(temperature_arr, arr_size)
+  temperature_arr.shift(arr_size)
+end
+
+def average(temp, size)
+  (temp.inject(0, :+) / size.to_f).ceil(1)
+end
+
+table = parse_csv_data('test.csv')
+dates = table.by_col[0]
+degrees = table.by_col[1]
+degrees_to_i = parse_to_int(degrees)
+
+august_dates_size = month_dates_length(dates, /^.{5}08.{3}/)
```

Comment:
your solution doesn't seem to be universal. If we would have a bigger CSV with some other months - it wouldn't work. 
Usually, when you implement this kind of functionality - you should consider all edge cases, including having a different CSV. As for right now I can accept this HW, as you might wouldn't have enought time to fix this issue. But for future, please make sure you are implementing universaly working functionality, instead of having a code which work only for particular dataset.

Reply:
you might need some kind of loop here, which will go and set a hash key as a month number, and value as min, max, average. or something like that.